Remove `--govuk-frontend-breakpoint` CSS properties (#6385)
alphagov/govuk-frontend: CHANGELOG.md

This change was introduced in [pull request #6366: Update Dart Sass tests minimum version to 1.79.0](https://github.com/alphagov/govuk-frontend/pull/6366)

#### Stop using the `--govuk-frontend-breakpoint` CSS custom properties

> CSS custom properties starting `--govuk-frontend-breakpoint` were renamed to begin with `--govuk-breakpoint` in [GOV.UK Frontend 5.11.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.0). The old names for these properties have now been removed.

Reviewer: ```suggestion
We renamed CSS custom properties starting `--govuk-frontend-breakpoint` to begin with `--govuk-breakpoint` in [GOV.UK Frontend 5.11.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.0). We've now removed the previous names for these properties.
```
